Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,445 @@
+import {Grid, GridItem, Box, Flex, Spacer, Text, Center, useColorMode, VStack, Image, SimpleGrid} from "@chakra-ui/react";
+import HeadingCell from "../Shared/HeadingCell";
+import CONSTANTS from "../../data-constants.json"
+import getColor from "../Shared/colors";
+import images from "../Motor_Motion/Icons/Images"
+import RangeCell from "../Shared/RangeCell";
+
+export default function Motor_Motion(props) {
+    const { colorMode } = useColorMode();
+
+    const borderCol = getColor("border", colorMode);
+    const greenBgCol = getColor("ppc_mppt_greenBg", colorMode);
```

Comment:
For the red/green text and bg colors, let's change the name to be more generic (e.g. greenBg/redBg and txtGreenBg/txtRedBg) instead of leaving them prepended by ppc_mppt. This will have to change in PPC_MPPT and anywhere else it might be used.